components/BCRegistriesAccount.vue

```diff
@@ -416,6 +447,14 @@ export default class BCRegistriesAccount extends Vue {
     return getFeatureFlag('bcregistry-ui-bca-enabled') as boolean
   }
 
+  get showMhrFlag (): boolean {
+    return getFeatureFlag('bcregistry-ui-mhr-enabled') as boolean
```

Comment:
Are these added in LD?

Reply:
This was pre existing yessir, but wasn't in use anywhere that Patrick or myself could see. 

The beta flag was net new and added though, afaik.